neurom/io/hdf5.py

```diff
@@ -81,7 +81,7 @@ class H5(object):
     def read_v1(filename):
```

Comment:
I think it would be good to have an option to switch this duplicate removal on, set by default to True